docker/server/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:18.12.1
+
+# Create app directore
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
```

Comment:
we should actually prevent any accidental `package-lock.json` files from being added that could conflict with `yarn.lock`

Should we update to just `package.json`